buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -296,23 +296,25 @@ class BeamModulePlugin implements Plugin<Project> {
     def bigtable_version = "1.0.0"
     def bigtable_proto_version = "1.0.0-pre3"
     def google_clients_version = "1.23.0"
-    def google_auth_version = "0.7.1"
-    def grpc_version = "1.2.0"
-    def protobuf_version = "3.2.0"
+    def google_auth_version = "0.10.0"
+    def grpc_version = "1.13.1"
+    def protobuf_version = "3.6.0"
     def guava_version = "20.0"
-    def netty_version = "4.1.8.Final"
-    def grpc_google_common_protos = "0.1.9"
+    def netty_version = "4.1.25.Final"
+    def proto_google_common_protos = "1.12.0"
     def hamcrest_version = "1.3"
     def hadoop_version = "2.7.3"
     def jackson_version = "2.9.5"
     def spark_version = "2.3.1"
-    def pubsub_grpc_version = "0.1.18"
+    def spanner_grpc_version = "0.19.0"
```

Comment:
what is this used for and why such an old version ?

Reply:
i am also curious who would be responsible to manage these versions ? 

We had a recent omg/9825 (b/111221386) due to out of date grpc version in DF causing issues running a DF job involving spanner .. 
is there a plan to resolve this in DF and maintain version compatibility for all services ?

Reply:
This isn't the version of grpc, it is the version of this package: https://repo1.maven.org/maven2/com/google/api/grpc/grpc-google-cloud-spanner-v1/

It was release 2018-07-12. This package is currently built on every release of google-cloud-java.

Reply:
> is it possible to rename this to grpc-google-cloud-spanner-v1 ?

or atleast add a comment what it points to

Reply:
I could just remove the variable and put the version directly where it's used, since it is used in only one place.

DF is creating new tools/processes to pull in client updates more frequently. @lukecwik can answer that question or redirect to someone else.